generators/app/templates/infrastructure/package.json

```diff
@@ -23,23 +23,24 @@
   "dependencies": {
     "@apollo/client": "^3.5.6",
     "@axa-fr/react-oidc-context": "3.1.6",
-    "@bit/totalsoft_oss.react-mui.kit.core": "^2.9.55",
-    "@bit/totalsoft_oss.react-mui.styles": "^2.9.55",
-    "@bit/totalsoft_oss.react-mui.themes.blue-theme": "^2.9.55",
-    "@bit/totalsoft_oss.react-mui.themes.default-theme": "^2.9.55",
-    "@bit/totalsoft_oss.react-mui.themes.green-theme": "^2.9.55",
-    "@bit/totalsoft_oss.react-mui.themes.orange-theme": "^2.9.55",
-    "@bit/totalsoft_oss.react-mui.themes.red-theme": "^2.9.55",
-    "@bit/totalsoft_oss.react-mui.themes.vivid-orange-theme": "^2.9.55",
-    "@bit/totalsoft_oss.react-mui.themes.light-blue-theme": "^2.9.55",
-    "@material-ui/core": "^4.12.2",
-    "@material-ui/icons": "^4.11.2",
+    "@emotion/react": "^11.9.3",
+    "@emotion/styled": "^11.9.3",
+    "@mui/material": "^5.8.6",
+    "@mui/styles": "^5.8.6",
+    "@mui/icons-material": "5.8.4",
+    "@mui/lab": "5.0.0-alpha.96",
+    "@mui/x-date-pickers": "5.0.0-beta.6",
+    "@testing-library/react": "^12.1.2",
+    "@testing-library/react-hooks": "8.0.1",
     "@totalsoft/change-tracking-react": "^5.0.22",
     "@totalsoft/pure-validations-react": "^5.0.22",
     "@totalsoft/rules-algebra-react": "^5.0.22",
     "@totalsoft/validations": "^1.0.4",
+    "@totalsoft_oss/rocket-ui.core": "^0.0.8",
+    "@totalsoft_oss/rocket-ui.themes": "^0.0.15",
     "apollo-link-mock": "^1.0.1",
     "apollo-upload-client": "^17.0.0",
+    "chart.js": "3.8.0",
```

Comment:
I would remove the following packages `chart.js` and `react-chartjs-2` from the generator, and from Chart component's peerDependencies if it is so, and move them as dependencies in Chart components in bit.